README.md

```diff
@@ -1,2 +1,142 @@
-# vpsearch
-Microbiome code for opensourcing
+----------------------------------------------------------------
+vpsearch - Fast Vantage-Point Tree Search for Sequence Databases
+----------------------------------------------------------------
+
+This is a package for indexing and querying a sequence database for fast
+nearest-neighbor search by means of vantage point trees. For reasonably large
```

Comment:
Should probably have some language here that clarifies that this is for comparing sequences whole-to-whole, not subsequence search a la BLAST. I don't quite have the most felicitous wording for that, but it's an important thing to have up front.

Reply:
@rkern I've added a note [here](https://github.com/enthought/vpsearch/pull/2/commits/cf6ef2a8c31c65ade76214848525a0a472030b18), let me know if you think this is sufficient.